### PR TITLE
Plat 4725/filters crd

### PR DESCRIPTION
**Pull request by naemono**

### CODE REVIEW

File: pkg/apis/objectrocket/v1beta1/filter.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// SensuEventFilterList is a list of sensu assets.
+type SensuEventFilterList struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard list metadata
+	// More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []SensuAsset `json:"items"`
```
Comment (amock) [inline pkg/apis/objectrocket/v1beta1/filter.go]:
```suggestion
	Items           []SensuEventFilter `json:"items"`
```

File: pkg/controller/informer.go [hunk 2]
```diff
@@ -173,6 +180,12 @@ func (c *Controller) run() {
 		for c.processNextHandlerItem() {
 		}
 	}()
+	go func() {
+		defer wg.Done()
+		defer c.informers[api.SensuEventFilterResourcePlural].queue.ShutDown()
+		for c.processNextEventfilterItem() {
```
Comment (jonsabo) [inline pkg/controller/informer.go]:
```suggestion
		for c.processNextEventFilterItem() {
```

File: pkg/controller/informer.go [hunk 3]
```diff
@@ -305,6 +318,40 @@ func (c *Controller) processNextHandlerItem() bool {
 	return true
 }
 
+func (c *Controller) processNextEventfilterItem() bool {
```
Comment (jonsabo) [inline pkg/controller/informer.go]:
```suggestion
func (c *Controller) processNextEventFilterItem() bool {
```

**Comment by jonsabo** [comment]
I only have nits, otherwise LGTM

**Pull request merged**